Bug fixing:

alex-zinin opened the pull request:
1)немного рефакторнуть сервисы, добавив везде обработку нулевых айди
   2)сделать файлы с конфигурацией базы и её иницилизирующими данными, в идеале сделать это в докере
   3)во всех компонентах либо оставить autowired над конструкторами, либо везде убрать

Commit:
Bug fixing:
1)немного рефакторнуть сервисы, добавив везде обработку нулевых айди
   2)сделать файлы с конфигурацией базы и её иницилизирующими данными, в идеале сделать это в докере
   3)во всех компонентах либо оставить autowired над конструкторами, либо везде убрать

FilippovIvan19 commented on knownet-application/src/main/resources/application.properties:
```diff
@@ -1,7 +1,7 @@
 server.port=8083
 
-logging.level.com=INFO
+logging.level.com=DEBUG
```
не надо комитить, а тем более пушить такие изменения

alex-zinin replied:
Сории

FilippovIvan19 commented on knownet-web/src/main/java/com/ncedu/knownetweb/LessonController.java:
```diff
@@ -26,12 +26,14 @@ public LessonController(LessonService lessonService) {
     @GetMapping("/all")
     public ResponseEntity<List<Lesson>> findAll() {
         log.debug("requested: lessons get    (all)");
+        
```
в это файле убери все изменения, они тоже отладочные, не надо их сохранять

alex-zinin replied:
Готово

FilippovIvan19 replied:
нет, строчка лишняя, сравнивай файлы. не надо делать изменения, которые ничего не несут, чтобы не вносить неразбериху

alex-zinin replied:
Готово

Commit:
Bug_fix
1) changing log level
2) cancelling changing in LessonController

FilippovIvan19 commented on knownet-web/src/main/java/com/ncedu/knownetweb/LearnRequestController.java:
```diff
@@ -16,7 +17,8 @@
     private final LearnRequestService learnRequestService;
-    
+
+    @Autowired
```
если решил добавлять Autowired, то над конструкторами сервисов тоже добавь

alex-zinin replied:
Готово

Commit:
Bug_fix
1) LessonService: cancelling changes
2) @Autowired adding

Commit:
Bug_fix
1) sql conf

Commit:
Bug_fix
1) id auto increment in database conf files

FilippovIvan19 commented on knownet-web/src/main/java/com/ncedu/knownetweb/LessonController.java:
```diff
@@ -53,7 +53,7 @@ public LessonController(LessonService lessonService) {
         return ResponseEntity.ok().body(lessonService.findByTeacherId(teacherId));
     }
-    
+
```
добавь 4 пробела

FilippovIvan19 replied:
или я немного подзапутался. короче надо сделать так, чтобы это файл не отличался, потому что в нём нет изменений и его не надо изменять в новой версии

FilippovIvan19 commented on knownet-web/src/main/java/com/ncedu/knownetweb/LessonController.java:
```diff
@@ -94,5 +94,5 @@ public LessonController(LessonService lessonService) {
     }
-    
-}
+
```
добавь 4 пробела

Commit:
Bug_fix
1) lesson controller such as in master
2) configuration_file fixed(serial to integer, request fix)
3) id != null fix and log.warn added
